packages/core/src/common/errors.ts

```diff
@@ -26,6 +26,11 @@ export const HOTKEYS_WARN_DECORATOR_NO_METHOD = ns + ` @HotkeysTarget-decorated
 export const HOTKEYS_WARN_DECORATOR_NEEDS_REACT_ELEMENT =
     ns + ` "@HotkeysTarget-decorated components must return a single JSX.Element or an empty render.`;
 
+export const ICON_STRING_NAMES_NOT_SUPPORTED =
```

Comment:
is this used in `core` package? pretty sure it's safe to delete as it's defined in `icons` below.